src/components/profile/ChangePasswordForm.tsx

```diff
@@ -56,3 +60,8 @@ export const ChangePasswordForm = ({
     )}
   </Formik>
 );
+const styles = StyleSheet.create({
+  formInput: {
+    backgroundColor: '#fff',
+  },
+});
```

Comment:
Not sure why this is needed. We should replace all text fields with native-base components and this should be fine.
As I already mentioned we should avoid hard-coded hex values and only native-base color constants should be used.